openfe/setup/atom_mapping/lomap_scorers.py

```diff
@@ -291,10 +291,11 @@ def creates_heterocyle(mol):
 
 def transmuting_methyl_into_ring_score(mapping: LigandAtomMapping,
```

Comment:
Sorry to ask this, but could you add a direct test for this here? It'll help document intentions in the future.